.github/workflows/auto-pr.yaml

```diff
@@ -0,0 +1,97 @@
+name: Auto-PR
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - closed
+
+env:
+  TERM: dumb
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.CR_PAT }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # This is necessary to avoid unexpected auto-merge in git cherry-pick
+          fetch-depth: 0
+
+      - name: Create pull requests
+        run: |
+          # Get the author of the merged PR
+          assignee=$(gh api graphql -F owner=${{ github.event.repository.owner.login }} -F repoName=${{ github.event.repository.name }} -F userName=${{ github.event.pull_request.user }} -f query='
+          query($owner: String!, $repoName: String!, $userName: String!) {
+            repository(owner: $owner, name: $repoName) {
+              assignableUsers(first: 100, query: $userName) {
+                nodes {
+                  name,
+                  email
+                }
+              }
+            }
+          }' | jq -r '.data.repository.assignableUsers.nodes[0]'
+          )
+
+          # Get GitHub projects associated with the merged PR and extract full versions (e.g. "3.7.1")
+          versions=$(gh api graphql -F owner=${{ github.event.repository.owner.login }} -F repoName=${{ github.event.repository.name }} -F pullRequestId=${{ github.event.number }} -f query='
+          query($owner: String!, $repoName: String!, $pullRequestId: Int!) {
+            repository(owner: $owner, name: $repoName) {
+              pullRequest(number: $pullRequestId) {
+                projectsV2(first: 100) {
+                  nodes {
+                    title
+                  }
+                }
+              }
+            }
+          }
+          ' | jq -r '.data.repository.pullRequest.projectsV2.nodes[].title' | sed -n 's/^ScalarDB \([0-9]*\.[0-9]*\.[0-9]*\).*/\1/p'
+          )
+          echo -------------
+          echo "versions: $versions"
+          echo -------------
+
+          # Extract target support branch and release branch names from the full versions
+          branches=$(echo $versions | ruby -e '
+            require "set"
+
+            versions = STDIN.read.strip.split
+
+            versions.inject(Set.new) {|acc, v|
```

Comment:
According to a discussion elsewhere, a bit more logic is needed (e.g. branch `3` doesn't permanently exist actually.) I'll update this logic later.